src/components/organisms/Accordion/Accordion/Accordion.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Accordion from '..';
+
+describe('<Accordion />', () => {
+  it('should render aria-label', () => {
+    const ariaLabel = 'test-accordion';
+    const { queryByText } = render(<Accordion aria-label={ariaLabel} />);
+    expect(queryByText).toBeTruthy();
```

Comment:
Hmm, this doesn't seem right? 🤔 
( `queryByText` is always _truthy_ as it's a function 😅 )